hermes/src/lib.rs

```diff
@@ -150,7 +155,7 @@ pub trait SoundFeedbackBackendFacade: IdentifiableToggleableBackendFacade {}
 pub trait AsrFacade: ComponentFacade + ToggleableFacade {
     fn publish_start_listening(&self, start: AsrStartListeningMessage) -> Fallible<()>;
     fn publish_stop_listening(&self, site: SiteMessage) -> Fallible<()>;
-    fn publish_reload(&self) -> Fallible<()>;
+    fn publish_reload(&self, reload: LoadMessage) -> Fallible<()>;
```

Comment:
use a separate `ReloadRequestMessage`, with a single field `id`, the `component` in `LoadMessage` is useless and missleading here